backend/models.py

```diff
@@ -209,30 +210,30 @@ class Receipt(models.Model):
     purchase_category = models.CharField(max_length=200, blank=True, null=True)
 
     class Meta:
-        constraints = [USER_OR_ORGANIZATION_CONSTRAINT()]
+        constraints = [user_or_organization_constraint()]
 
     def __str__(self):
         return f"{self.name} - {self.date} ({self.total_price})"
 
-    def has_access(self, user: User) -> bool:
+    def has_access(self, user: User):
         if not user.is_authenticated:
             return False
-
-        if user.logged_in_as_team:
-            return self.organization == user.logged_in_as_team
-        else:
-            return self.user == user
+        return self.organization == user.logged_in_as_team if user.logged_in_as_team else self.user == user
 
 
 class ReceiptDownloadToken(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     file = models.ForeignKey(Receipt, on_delete=models.CASCADE)
     token = models.UUIDField(default=uuid4, editable=False, unique=True)
 
+    class Meta:
+        verbose_name = "Receipt Download Token"
+        verbose_name_plural = "Receipt Download Tokens"
+
 
 class Client(models.Model):
-    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
These need to be left as blank=True otherwise you can't edit or create a Client properly